src/Bicep.Core.UnitTests/Utils/ServiceBuilderExtensions.cs

```diff
@@ -49,6 +50,9 @@ public static ServiceBuilder WithEmptyAzResources(this ServiceBuilder serviceBui
     public static ServiceBuilder WithWorkspaceFiles(this ServiceBuilder serviceBuilder, IReadOnlyDictionary<Uri, string> fileContentsByUri)
         => serviceBuilder.WithRegistration(x => x.WithWorkspaceFiles(fileContentsByUri));
 
+    public static ServiceBuilder WithEnvironmentVariables(this ServiceBuilder serviceBuilder, params (string key, string? value)[] variables)
+        => serviceBuilder.WithRegistration(x => x.WithEnvironmentVariables(variables));
+
```

Comment:
What is the purpose of this piece if we are ultimately calling the same function `WithEnvironmentVariables` again?

Reply:
As discussed, purely for reducing boilerplate - nothing functional.